zokrates_core/src/proof_system/mod.rs

```diff
@@ -46,15 +46,15 @@ pub type Fq = String;
 pub type Fq2 = (String, String);
 
 #[derive(Serialize, Deserialize)]
-pub struct G1Affine(Fq, Fq);
+pub struct G1Affine(pub Fq, pub Fq);
```

Comment:
Where is this needed?

Reply:
Oh my bad, I forgot to revert that change, I needed it before for a reason I don't remember anymore